front/src/styles/scss/applications/operationalStudies/_scenarioV2.scss

```diff
@@ -107,11 +107,44 @@
 
       .board-btn {
         padding: 1px 8px 3px 8px;
-        color: var(--grey80);
-        font-size: 0, 750rem;
+        font-size: 0.750rem;
         line-height: 16px;
         border-radius: 4px;
-        background-color: var(--ambientB5);
+
+        // START HANDLE BOARD STATE
+
+        // On state
+        &.on {
+          color: var(--grey80);
+          background-color: var(--ambientB5);
+
+          &:hover {
+            color: var(--black100);
+            background-color: var(--white100);
+          }
+
+          &:active {
+            color: var(--grey50);
+            background-color: var(--ambientB15);
+          }
```

Comment:
You have inverted the active and hover style (mockup order: on rest, on hover, on active):
![image](https://github.com/user-attachments/assets/f512d481-299f-404a-85aa-69476a4b5c8f)

Same for the off state